examples/operator/auto_weighting.py

```diff
@@ -0,0 +1,58 @@
+"""Example demonstrating the usage of the ``auto_weighting`` decorator."""
```

Comment:
This needs to be somewhat fleshed out, especially given that new users come here

Reply:
Can do